Mathlib/Data/Finset/Basic.lean

```diff
@@ -2200,20 +2216,96 @@ theorem union_sdiff_self (s t : Finset α) : (s ∪ t) \ t = s \ t :=
   sup_sdiff_right_self
 #align finset.union_sdiff_self Finset.union_sdiff_self
 
+-- TODO: Do we want to delete this lemma and `finset.disj_union_singleton`,
+-- or instead add `finset.union_singleton`/`finset.singleton_union`?
 theorem sdiff_singleton_eq_erase (a : α) (s : Finset α) : s \ singleton a = erase s a := by
   ext
   rw [mem_erase, mem_sdiff, mem_singleton, and_comm]
 #align finset.sdiff_singleton_eq_erase Finset.sdiff_singleton_eq_erase
 
+-- This lemma matches `finset.insert_eq` in functionality.
```

Comment:
as above. Perhaps run `fix_comments.py`?

Reply:
I just tried it and it missed "`finset.union_singleton`/`finset.singleton_union`".